hudi-cli/conf/hudi-env.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# Set the necessary environment variables
+export HADOOP_CONF_DIR=/etc/hadoop/conf
```

Comment:
while we are here, allow for overriding `HADOOP_CONF_DIR` and `SPARK_CONF_DIR` by setting the value outside the script?

Reply:
> while we are here, allow for overriding `HADOOP_CONF_DIR` and `SPARK_CONF_DIR` by setting the value outside the script?

Thanks for review, changed it to `${HADOOP_CONF_DIR:-/etc/hadoop/conf}`.